eng/testing/outerBuild.targets

```diff
@@ -1,17 +1,6 @@
 <Project>
-  <Target Name="Test">
-    <ChooseBestTargetFrameworksTask BuildTargetFrameworks="$(BuildTargetFramework)-$(OSGroup)"
-                                    SupportedTargetFrameworks="$(TargetFrameworks)"
-                                    RuntimeGraph="$(RuntimeGraph)" >
-      <Output TaskParameter="BestTargetFrameworks" ItemName="_BestTargetFramework" />
-    </ChooseBestTargetFrameworksTask>
-
-    <AddTargetFrameworksToProjectTask ProjectName="$(MSBuildProjectFile)"
-                                      BestTargetFrameworks="@(_BestTargetFramework)" >
-      <Output TaskParameter="InnerBuildProjects" ItemName="TestProject" />
-    </AddTargetFrameworksToProjectTask>
-    
-    <MSBuild Projects="@(TestProject)"
+  <Target Name="Test" DependsOnTargets="GetProjectWithBestTargetFrameworks">    
```

Comment:
Same here, can you inline that file instead please?

Reply:
Same reason as described above